arviz/stats/diagnostics.py

```diff
@@ -311,7 +370,7 @@ def rhat(data, *, var_names=None, method="rank"):
 
 
 def mcse(data, *, var_names=None, method="mean", prob=None):
-    """Calculate Markov Chain Standard Error for statistic.
+    """Calculate Markov Chain Standard Error statistic.
```

Comment:
it'd be cool to have a reference for this eventually (but certainly not needed in this PR)